### DISCUSSION TITLE
关于json schema改变以后但是视图不刷新的问题。

### DISCUSSION THREAD

**Discussion by xiaosatufu** [alibaba/formily#2853] [Q&A 2.x]
refreshUiSchema() {
      let newSchema = JSON.parse(JSON.stringify(this.schema))
      this.schema = Object.assign({}, newSchema)
    },
通过次方法变更schema的内容，数据发生了变化，视图没有改变。
如果从控制台复制并直接赋值给schema可成功渲染，不知道动态渲染是怎么处理，官方文档表单受控，只有react的模式，请问有vue的表单受控相关文档吗？

**Comment by laibao101** [comment]
schema变化后，调用createForm重新创建一个form能解决你的问题
